client/src/components/TimetableSwitcher/TimetableSwitcher.tsx

```diff
@@ -2,8 +2,10 @@ import React, { useState, memo } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Dropdown, ButtonGroup, Button } from 'react-bootstrap';
 import {
-	FaTrashAlt, FaPlusSquare, FaPen, FaCopy,
+	FaTrashAlt, FaPlusSquare, FaPen, FaCopy, FaDownload, FaFileDownload
 } from 'react-icons/fa';
+import domtoimage from 'dom-to-image';
```

Comment:
Could be renamed to `domToImage` to stick to naming conventions.
```suggestion
import domToImage from 'dom-to-image';
```